### ISSUE TITLE
Not possible to share project documents with non-portal users

### ISSUE THREAD

**Issue by dokterbob** [ONLYOFFICE/CommunityServer#150]
From within the Documents app, a document can be shared with external users using a link:
![ssp_temp_capture](https://user-images.githubusercontent.com/22742/55540944-e0afc700-56bb-11e9-9d0d-20f7a173c393.png)
![ssp_temp_capture](https://user-images.githubusercontent.com/22742/55541151-5320a700-56bc-11e9-92c4-eb6dcb07a90d.png)

However, this feature is not available from within Project Documents, which is confusing to the user:
![ssp_temp_capture](https://user-images.githubusercontent.com/22742/55541186-616ec300-56bc-11e9-82b4-947535cacbb3.png)

**Comment by Carazyda** [comment]
@dokterbob Yes, sharing settings is not allowed in project documents. We work on this feature, and it will be available in next versions.

**Comment by dokterbob** [comment]
Great to hear you’re working on it! Nothing’s perfect from the start. ;)

**Comment by luzz94** [comment]
@dokterbob any news about this feature? it was dropped?
